### DISCUSSION TITLE
vi/models/yolo-world/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#17740] [Docs]
# vi/models/yolo-world/

Khám phá YOLO -Mô hình thế giới để phát hiện đối tượng từ vựng mở hiệu quả, thời gian thực bằng cách sử dụng Ultralytics YOLOv8 tiến bộ. Đạt được hiệu suất cao nhất với tính toán tối thiểu.

https://docs.ultralytics.com/vi/models/yolo-world/

**Comment by phanductri** [comment]
Mình thấy trên kênh này cũng có nói về YOLO model và code hay nè: https://www.youtube.com/@VIETECH168/videos

**Reply by glenn-jocher** [reply]
@phanductri thank you for sharing the video channel suggestion, but we recommend referring to the official YOLO-World documentation for accurate, detailed guidance on using the models: https://docs.ultralytics.com/models/yolo-world. Let us know if you have any specific questions!

**Comment by UltralyticsAssistant** [comment]
👋 Hello, thank you for engaging with the Ultralytics community 🚀! It seems you're diving into YOLO with the exciting new yolo-world model. For insights and guidance, the updated [Docs](https://docs.ultralytics.com/vi/models/yolo-world/) are a great place to start — check them out for information on effective real-time open vocabulary object detection using Ultralytics YOLOv8.

If you're encountering a 🐛 Bug, kindly help us with a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) so an Ultralytics engineer can address it promptly.

For custom training inquiries and general questions, providing comprehensive details such as dataset samples, configuration settings, and logs will ensure we can best assist you. Additionally, our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) might come in handy.

Join our thriving community to connect further: [Discord](https://ultralytics.com/discord) for live chats, [Discourse](https://community.ultralytics.com) for in-depth threads, or engage with us on [Reddit](https://reddit.com/r/ultralytics) 💬.

## Upgrade

Make sure to upgrade to the latest `ultralytics` package with all necessary [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) installed, ensuring the issue doesn't persist in the latest version:

```bash
pip install -U ultralytics
```

## Environments

Our tools are compatible with a range of environments. Here’s where you can quickly get started:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM with the [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI with the [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. Follow the [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

This badge indicates the current status of [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests. A green badge means all tests for YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) are passing on macOS, Windows, and Ubuntu. 

An Ultralytics engineer will be along soon to further assist. 🤖🔍
